conf/metrics.properties.template

```diff
@@ -0,0 +1,9 @@
+
+# Enable StatsdSink for all instances by class name
+*.sink.statsd.class=org.apache.spark.metrics.sink.StatsDSink
+*.sink.statsd.prefix=spark
```

Comment:
It would be nice to re-use the `DCOS_SPACE` env. var (if defined, which it _should_ be) as the `statsd.prefix` since it will tell us where in the DC/OS Namespace (App ID/Group) these reside.

Reply:
Would that appear as a tag in dcos-metrics? I thought it automatically included e.g. `framework_id`, `framework_name`, ... (used to do that, at least)

Reply:
It doesn't show up in the prefix, which is hard-coded to "spark". It shows up in the "task_name" tag. For example, here is an event from the "spark-7" service (`task_name="spark-7"`):

> spark_mesos_cluster_launcheddrivers{cluster_id="91dcf789-cb74-4c28-8089-8f11f426bc73",container_id="787cf2f6-040d-40bb-aaf3-2b5bdd18bd11",framework_id="215aa77b-8918-46de-98f2-20e87066d591-0001",hostname="172.31.5.165",instance="172.31.5.165:61091",job="dcos",mesos_id="215aa77b-8918-46de-98f2-20e87066d591-S17",task_id="spark-7.9990c5fe-4c95-11e8-8d35-3291aaa04f72",task_name="spark-7"}